integration/kubernetes/helm-chart/alluxio/templates/fuse/daemonset.yaml

```diff
@@ -99,19 +121,8 @@ spec:
               memory: {{ .Values.fuse.resources.requests.memory }}
             {{- end }}
           {{- end }}
-          command: ["/entrypoint.sh"]
```

Comment:
Revert this accident modification code

Reply:
> Revert this accident modification code
Thanks for review , I have reversed this accident modification code